.github/workflows/checks.yml

```diff
@@ -25,3 +25,6 @@ jobs:
 
       - name: Build
         run: bun run build
+
+      - name: Pinned Dependencies Check
+        run: bun ./bin/index.js
```

Comment:
Though it's not a perfectly exhaustive check, this can help prevent future releases from shipping a broken executable.

And plus, it will guarantee the dependencies are pinned in this repo!

Reply:
love it... thanks @jeremybanka